### PR TITLE
8bit wanderers

### PR DESCRIPTION
**Pull request by pstalcup**

### CODE REVIEW

**Comment by horrible-little-slime** [comment]
I think we need a way of evaluating the opportunity cost of losing a slot. On Embezzlers that can matter a lot--for free fights, not so much

**Comment by horrible-little-slime** [comment]
per discussion in dm, we are ignoring the opportunity cost of the slot by simply making embezzlers reject locations with equipment requirements

which isn't great, but that's life baby

File: src/wanderer/eightbit.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+function pointsPerTurn(color: BonusColor) {
+  const current = get("8BitColor") === color;
+  const addedBonus = current ? 100 : 50;
+  const denominator = current ? 20 : 10;
+  const rawPoints = clamp(
+    numericModifier(modifierForColor[color]) - minimumForPoints[color],
+    0,
+    300,
+  );
```
Comment (horrible-little-slime) [inline src/wanderer/eightbit.ts:44]: There's a mafia

Comment (pstalcup) [reply src/wanderer/eightbit.ts:44]: what is it

Comment (horrible-little-slime) [reply src/wanderer/eightbit.ts:44]: number [eightBitPoints](https://wiki.kolmafia.us/index.php?title=eight_bit_points)(Location)
number [eightBitPoints](https://wiki.kolmafia.us/index.php?title=eight_bit_points)(Location, string, number)

**Review by Tokoeka — DISMISSED** [review]

File: src/barfTurn.ts [hunk 2]
```diff
@@ -309,7 +318,10 @@ const turns: AdventureAction[] = [
 
       propertyManager.setChoices(wanderer.getChoices("yellow ray"));
       const location = wanderer.getTarget("yellow ray");
-      freeFightOutfit({}, { location, allowAttackFamiliars: !usingDuplicate }).dress();
+      freeFightOutfit(
+        { equip: wanderer.getEquipment("yellow ray") },
```
[the remark this answers is not in the document]
Comment (horrible-little-slime) [reply src/barfTurn.ts:322]: we can do that in lieu of passing a `target` or `location` or whatever

File: src/tasks/daily.ts [hunk 3]
```diff
@@ -398,6 +398,23 @@ const DailyTasks: GarboTask[] = [
...
+      // taken from autoscend
+      visitUrl("place.php?whichplace=woods");
+      visitUrl("place.php?whichplace=forestvillage&action=fv_mystic");
+      visitUrl(
+        "choice.php?pwd=&whichchoice=664&option=1&choiceform1=Sure%2C+old+man.++Tell+me+all+about+it.",
+      );
+      visitUrl(
+        "choice.php?pwd=&whichchoice=664&option=1&choiceform1=Against+my+better+judgment%2C+yes.",
+      );
+      visitUrl("choice.php?pwd=&whichchoice=664&option=1&choiceform1=Er,+sure,+I+guess+so...");
```
Comment (horrible-little-slime) [inline src/tasks/daily.ts:415]: I feel like these should probably be `runChoice`s

File: src/tasks/daily.ts [hunk 4]
```diff
@@ -398,6 +398,23 @@ const DailyTasks: GarboTask[] = [
     },
     limit: { skip: 3 }, // Sometimes need to cycle through some dialogue
   },
+  {
+    name: "Continuum Transfunctioner",
+    ready: () => canAdventure($location`The Spooky Forest`),
+    completed: () => have($item`continuum transfunctioner`),
+    do: (): void => {
+      // taken from autoscend
+      visitUrl("place.php?whichplace=woods");
```
Comment (horrible-little-slime) [inline src/tasks/daily.ts:407]: I would describe myself as 80% sure we don't need this first one

File: src/wanderer/eightbit.ts [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  blue: $location`Megalo-City`,
+  green: $location`Hero's Field`,
+  red: $location`The Fungus Plains`,
+};
+
+function value(color: BonusColor) {
+  const denominator = globalOptions.ascend
+    ? get("8BitScore") - FAT_LOOT_TOKEN_COST
+    : FAT_LOOT_TOKEN_COST;
+  return (garboValue($item`fat loot token`) * eightBitPoints(locationColor[color])) / denominator;
```
Comment (horrible-little-slime) [inline src/wanderer/eightbit.ts:26]: pixel rock is 16k; is it worth accounting for that?
You can declare it out of scope if you want

Comment (horrible-little-slime) [reply src/wanderer/eightbit.ts:26]: out of scope for this round of implementation

**Review by horrible-little-slime — DISMISSED** [review]

**Pull request merged**